api/guides/access_tokens.md

```diff
@@ -1,133 +1,100 @@
 
-Give it a descriptive **Name** and a detailed **Description** for others to easily identify this application and its purpose
+Once you have created a new project, click **Add API** on the **Project Overview** screen.
 
-#### Generate a certificate
+![](/images/access-token/add-api-button.png)
 
-Next, generate an SSL private key and a public certificate.
+The **Add an API** screen appears. Select **Experience Platform Launch API** from the list of available APIs before clicking **Next**.
 
-##### macOS & Linux / UNIX
+![](/images/access-token/add-launch-api.png)
 
-Open a terminal application (Terminal.app, for example). Using OpenSSL, generate a new SSL certificate and key.
-```bash
-openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout private.key -out adobe-io-public.crt
-```
+On the next screen, you are prompted to create a JWT credential be either generating a new keypair or uploading your own public key. For this tutorial, select the **Generate a key pair** option, then select **Generate keypair** in the bottom-right corner.
 
-##### Windows
+![](/images/access-token/create-jwt.png)
 
-Download an OpenSSL client to generate public certificates; For example, [OpenSSL Windows client.](https://bintray.com/vszakats/generic/download_file?file_path=openssl-1.1.1-win64-mingw.zip)
+The next screen confirms the keypair has successfully generated, and a compressed folder containing a public certificate and a private key is automatically downloaded to your machine. You will need this private key in a later step to generate an access token.
 
-Extract the folder and copy it to `C:/libs`
+Select **Next** to continue.
 
-Open a command line window and execute the following:
+![](/images/access-token/keypair-generated.png)
 
-```bash
-set OPENSSL_CONF=C:/libs/openssl-1.1.1-win64-mingw/openssl.cnf
-cd C:/libs/openssl-1.1.1-win64-mingw/
-openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout private.key -out adobe-io-public.crt
-```
-
-These commands will create two files in the current directory _(usually the current user's home directory)_.
+The next screen prompts you to select the product profile(s) to associate with the API integration.
 
 {% alert info, Note %}
-The names of the files may be changed as desired. Keep the private key safe and limit access!
+Product profiles are managed by your organization through the Adobe Admin Console, and contain specific sets of permissions for granular features in Adobe Experience Platform. Product profiles and their permissions can only be managed by users with administrator privileges within your organization. If you are unsure which product profile(s) to select for the API, contact your administrator.
```

Comment:
Maybe say "Adobe Experience Platform Launch" in the first sentence?  Instead of just "Adobe Experience Platform".